Box Quick Update (#39267)
demisto/content: Packs/Box/README.md

4. The **Account** tab should open by default. Locate the **General Options** section.
5. Select your preferred timezone from the pulldown menu under **Time Zone**.
6. Click **Save Changes** in the top right to save your settings.

## Enabling Box Event Collector

> In order to configure the Box Event Collector to receive log messages:

Reviewer: ```suggestion
To configure the Box Event Collector to receive log messages:
```

> 1. Make sure you have the Box pack installed on your XSIAM tenant.

Reviewer: ```suggestion
1. Make sure you have the Box pack installed on your Cortex XSIAM tenant.
```

> 2. Go to **Settings** &rarr; **Configurations** &rarr; **Automation & Feed Integrations**
> 3. At the search bar, type **Box** and press on **+ Add instance**

Reviewer: ```suggestion
2. Go to **Settings** &rarr; **Configurations** &rarr; **Automation & Feed Integrations**.
3. In the search bar, type **Box** and click **+ Add instance**.
```

> 4. Follow the integration steps in order to send logs from Box to your XSIAM tenant.

Reviewer: ```suggestion
4. Follow the integration steps to send logs from Box to your Cortex XSIAM tenant.
```
Where are the integration steps?
